### DISCUSSION TITLE
Partners

### DISCUSSION THREAD

**Discussion by MicleSergiu** [sebastiano123-c/DroneIno#1] [General]
Hi! I am a student at computer science and I would like to work with you at this project to improve this togheter.

I'm making a drone for my final year project and I want to make a flight controler based on ESP32 and this software seems to be great already.

**Comment by sebastiano123-c** [comment]
Dear Sergiu,

I am sorry for the late answer I was out for work.
I am very glad that my project is useful for you.
To be honest I am currently doing other stuff right now and I don't think I
will work on it in the near future.
But I think that you can use it and if you find some errors please let me
know.
If you find some new features we can discuss them.
Please note that you can integrate DroneIno with another ESP32 CAM for live
video, telemetry and SD data storing.
This is described in my other repo
https://github.com/sebastiano123-c/Esp32-cam-telemetry.
Also you can find a detailed documentation for DroneIno here
https://github.com/sebastiano123-c/DroneIno/wiki.
Please note that my project is based on Joop Brokking's arduino project. He
has done a series of very useful videos that I reccomend you to watch here
https://www.youtube.com/watch?v=EeJeSE_5rns&list=PL0K4VDicBzshwCpUHzIB6hOLQVkDFHbxC
.
I hope you enjoy this project that gave me a lot of fun, and I hope you
find it useful for your work.
If you have any issue or fix feel free to ask me.
Bests,

Sebastiano

Il Gio 11 Mag 2023, 23:18 MicleSergiu ***@***.***> ha scritto:
